apps/tlon-mobile/src/components/AuthenticatedApp.tsx

```diff
@@ -19,11 +19,14 @@ function AuthenticatedApp({ initialNotificationPath }: AuthenticatedAppProps) {
   useDeepLinkListener();
 
   useEffect(() => {
-    configureClient(ship ?? '', shipUrl ?? '');
-    sync.start().catch((e) => {
-      console.warn('Sync failed', e);
-    });
-    useStorage.getState().start();
+    const start = () => {
```

Comment:
This wrapper function seems unnecessary? Or if it is necessary, I don't understand why, and a comment might help ha.